### PR TITLE
Match zfsonlinux for the check in dsl_scan_prefetch_cb().

### PR DESCRIPTION
**Pull request by kithrup**
I do not know if this would fix the panic, but it's clearly
a right change to make.

Ticket: #27524

### CODE REVIEW

**Comment by amotin** [comment]
Good catch, but any idea how could we miss this?  It seems to be so in the initial ZoL commit.  Was it some change they made during one of their rebases?  Can there be more surprises?

**Comment by amotin** [comment]
And could you still do something with your editor in respect of tab characters?  You are introducing code divergence without any reason.

**Comment by kithrup** [comment]
I went through the changes en masse.  It's likely I just skipped over this one because the field was being used differently before (it was a count of I/Os in progress, not bytes in progress).  I went through both zfsonlinux and us, looking for every use of that field.  Which turns out to only be in this file, so that part's all good now.
  
as for tabs:  damnit to make sure I had the exact same code I copy & pasted, and didn't try re-indenting.  Fixed/

**Comment by amotin** [comment]
I think I see one more critical divergence on line 1537:

spa->spa_scrub_inflight = BP_GET_PSIZE(&spic->spic_bp);

There should be += instead of just = .

**Comment by kithrup** [comment]
Yes, I blame that one on my eyes.  Went through it again and should have them all.  But i thought that before, so please keep telling me where I'm wrong 8-).

File: sys/cddl/contrib/opensolaris/uts/common/fs/zfs/dsl_scan.c [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/*
+4/*
```
Comment (amotin) [inline sys/cddl/contrib/opensolaris/uts/common/fs/zfs/dsl_scan.c]: Oops.

Comment (kithrup) [reply sys/cddl/contrib/opensolaris/uts/common/fs/zfs/dsl_scan.c]: Yeah found that by doing a build AFTER pushing.  Not doing that again!  (My control key is being wonky, as are a couple other keys.)  I've now built world & kernel and pushed it.

**Comment by kmoore134** [comment]
Jenkins: retest this please

**Comment by kithrup** [comment]
Okay, per my own testing and discussion with Alexander. merging this.

**Pull request merged**